mne/viz/epochs.py

```diff
@@ -166,7 +167,7 @@ def plot_epochs_image(epochs, picks=None, sigma=0., vmin=None,
     if combine is not None:
         ts_args["show_sensors"] = False
 
-    if picks is None:
+    if picks is None or not _is_numeric(picks):
```

Comment:
This is so that you don't get 300 epochs images if you do `epochs.plot_image(picks=["grad", "mag"])`